setup.cfg

```diff
@@ -1,6 +1,7 @@
 [flake8]
 min-version = 3.6
```

Comment:
Do we need to bump the minimum to 3.8?

Reply:
I think that can be done when beets officially moves the minimum supported version.